actions.yaml

```
@@ -0,0 +1,11 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+check-runners:
+    description: Get info on active and registered runners
+reconcile-runners:
+    description: Remove offline runners and replace any missing runners
```

Depending on how verbose a comment needs to be, I would suggest the following:
```suggestion
    description: Remove offline runners and replace any missing ones
```

---

Minor comments